tools/autopep8.sh

```diff
@@ -39,7 +39,7 @@ function main() {
     pip install -q virtualenv
   fi
 
-  GIT_COMMON_DIR=$(cd $(dirname $0) && git rev-parse --path-format=absolute --git-common-dir)
+  GIT_COMMON_DIR=$(git rev-parse --absolute-git-dir)
```

Comment:
The variable name is no longer correct, as this will correspond to `$GIT_DIR` and **not** to `$GIT_COMMON_DIR`. I think this should be fine, but could confirm that and update the name to reflect this?


> --absolute-git-dir
Like --git-dir, but its output is always the canonicalized absolute path.

Reply:
So is it a cosmetic issue, that we follow some convention for naming this issue?  Or, do you mean we want to make it overridable from the process environment, like we do for AUTOPEP8_VENV on the next line?

Reply:
It's a readability issue. It's like the difference between `PROJECT_SOURCE_DIR` and `CMAKE_SOURCE_DIR` in CMake. They're not the same thing and in edge cases the difference matters. Your commit introduces a behavioral change to the script, and the name of the `GIT_COMMON_DIR` variable is lying about what itself. It's not the git common dir anymore.

Reply:
> GIT_COMMON_DIR
If this variable is set to a path, non-worktree files that are normally in $GIT_DIR will be taken from this path instead. Worktree-specific files such as HEAD or index are taken from $GIT_DIR. See [gitrepository-layout[5]](https://git-scm.com/docs/gitrepository-layout) and [git-worktree[1]](https://git-scm.com/docs/git-worktree) for details. This variable has lower precedence than other path variables such as GIT_INDEX_FILE, GIT_OBJECT_DIRECTORY…​

Reply:
But, given that it's not being put into the environment, is GIT_DIR also not an appropriate name?

Reply:
I would be fine with `GIT_DIR`. Would shadowing the environment variable be harmful? The previous name was shadowing one, so it would be consistent with the current pattern to simply rename to `GIT_DIR` I think.